src/Files.App/Helpers/Dialog/DynamicDialogFactory.cs

```diff
@@ -58,7 +58,7 @@ public static DynamicDialog GetFor_ShortcutNotFound(string targetPath)
 			return dialog;
 		}
 
-		public static DynamicDialog GetFor_RenameDialog()
+		public static DynamicDialog GetFor_RenameDialog(string itemType)
```

Comment:
This dialog is only used when creating a new item, so the name “RenameDialog” is confusing and should be changed.

Reply:
I'd prefer doing this separately